src/module.ts

```diff
@@ -200,15 +200,15 @@ export default defineNuxtModule<ModuleOptions>({
 
       const configOptions = Object.keys(resolvedConfig)
       const template = addTemplate({
-        filename: 'tailwind.config.mjs',
-        getContents: () => `${configOptions.map(v => `import ${v} from "./tailwind.config/${v}.mjs"`).join('\n')}\nconst config = { ${configOptions.join(', ')} }\nexport { config as default, ${configOptions.join(', ')} }`
+        filename: 'tailwind.config/index.mjs',
+        getContents: () => `${configOptions.map(v => `import ${v} from "./${v}.mjs"`).join('\n')}\nconst config = { ${configOptions.join(', ')} }\nexport { config as default, ${configOptions.join(', ')} }`
```

Comment:
Already tried this as a solution, however it may cause `Failed to fetch dynamically imported module` in Vite. Using `write: true` (just for `tailwind.config/index.mjs`, not subpaths) worked.

Reply:
```suggestion
        filename: 'tailwind.config/index.mjs',
        getContents: () => `${configOptions.map(v => `import ${v} from "#build/tailwind.config/${v}.mjs"`).join('\n')}\nconst config = { ${configOptions.join(', ')} }\nexport { config as default, ${configOptions.join(', ')} }`,
        write: true
```